app/helpers/formatted-picker-date.js

```diff
@@ -26,6 +31,11 @@ function isYesterday(momentDate) {
   return momentDate.isSame(yesterday, 'd');
 }
 
+function isTomorrow(momentDate) {
+  let tomorrow = moment().add(1, 'days').startOf('day');
+  return momentDate.isSame(tomorrow, 'd');
+}
+
```

Comment:
this looks like duplicated code. wondering if we should consolidate the two implementations of `isTomorrow`?